### PR TITLE
[SQL] `az sql failover-group create/update`: Add parameter `--secondary-type` to support creation of geo/standby replica on partner server

### PR DESCRIPTION
**Pull request by saljain0101**
**Related command**
az sql failover-group create
az sql failover-group update

**Description**
Adds a new optional boolean parameter `--secondary-type` to command az sql failover-group create/update to enable creation of geo/standby replica on partner server while creating/updating sql failover group.

**Testing Guide**
Example1: az sql failover-group create --name fog --partner-server svr2 --resource-group saljain --server svr1 --add-db testdb 
--secondary-type Standby

Output: Creates a failover group named fog with testdb added as standby replica on svr2

Example2: az sql failover-group update --name fog --resource-group saljain --server svr1 --add-db testdb2 --secondary-type Geo

Output: Updates failover group named fog with testdb2 added as geo replica on svr2. Now fog has 2 databases: testdb with standby replica and testdb2 with geo replica on svr2

Example3: az sql failover-group create --name fog --partner-server svr2 --resource-group saljain --server svr1 --secondary-type Named

Output: az sql failover-group create: 'Named' is not a valid value for '--secondary-type'. Allowed values: Geo, Standby.

**History Notes**

[SQL] `az sql failover-group create/update`: Add parameter `--secondary-type` to support geo/standby replica on partner server

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/azure-cli/azure/cli/command_modules/sql/custom.py [hunk 1]
```diff
@@ -6425,8 +6430,9 @@ def failover_group_create(
         server_name,
         failover_group_name,
         partner_server,
+        secondary_type=None,
         partner_resource_group=None,
-        failover_policy=FailoverPolicyType.automatic.value,
+        failover_policy=FailoverPolicyType.manual.value,
```
Comment (evelyn-ys) [inline src/azure-cli/azure/cli/command_modules/sql/custom.py:6435]: Parameter's default value change is considered as breaking change. We only allow breaking changes twice a year and next breaking change window is Ignite Sprint

Comment (saljain0101) [reply src/azure-cli/azure/cli/command_modules/sql/custom.py:6435]: So, what should be the next steps? Should i revert the change or this PR will be part of next Nov release? I am fine with Nov release.

**Comment by yonzhan** [comment]
Please fix CI issues

**Pull request merged**